docs: add legacy server release notes (#7987)
microsoft/vscode-docs: remote-release-notes/v1_97.md

A new setting has been introduced to install extensions into a remote if that extension is already present locally. This mirrors the setting already available in the Dev Containers extension.

Configure with `setting(remote.SSH.defaultExtensionsIfInstalledLocally)`.

### Migration path for Linux legacy server

> We are fast approaching the end of support for [Linux legacy server](https://aka.ms/vscode-remote/faq/old-linux). VS Code release v1.98 (February 2025) will be the last release that supports these servers. As of release 1.99, you can no longer connect to these servers.

Reviewer: Maybe we can be specific:

> will be the last release that supports Linux servers with glibc < 2.28 or libstdc++ < 3.4.25.

> We have explored an option for the SSH extension where you provide a custom build of glibc and libstdc++ that enables connecting to the server. More details on this option is documented in this [draft](https://github.com/microsoft/vscode-docs/pull/7953/files) with test builds available [here](https://github.com/microsoft/vscode/pull/235232#issuecomment-2615764717).

Reviewer: Some alternative phrasing:

```suggestion
We are considering providing a workaround option for the SSH extension where you provide a custom build of glibc and libstdc++ that enables connecting to the server. More details can be found in this [draft](https://github.com/microsoft/vscode-docs/pull/7953/files) with test builds available [here](https://github.com/microsoft/vscode/pull/235232#issuecomment-2615764717).
```

> This workaround should provide additional time for migrating to a supported Linux distro, without having to disable VS Code updates. If you have any feedback on the workaround flow, comment [on this VS Code issue](https://github.com/microsoft/vscode/issues/231623).

Reviewer: ```suggestion
This workaround should provide additional time for migrating to a supported Linux distro, without having to disable VS Code updates. Please give us feedback [on this VS Code issue](https://github.com/microsoft/vscode/issues/231623).
```

A new setting has been introduced to install extensions into a remote if that extension is already present locally. This mirrors the setting already available in the Dev Containers extension.

Configure with `setting(remote.SSH.defaultExtensionsIfInstalledLocally)`.

> ### Migration path for legacy server users

Reviewer 2: Let's add Linux to the title to make it obvious. So

Migration path for legacy Linux server users

> ### Migration path for linux legacy server users

Reviewer 3: Prefer to drop the "users" part, but is minor. 
Make sure to uppercase Linux.

> As we are fast approaching the EOL for [legacy server](https://aka.ms/vscode-remote/faq/old-linux), February 2025 release v1.98 will be the last release supporting these servers. We have explored an option for the SSH extension to let users provide a custom build of glibc and libstdc++ required for connecting to our server. More details on this is documented in this [draft](https://github.com/microsoft/vscode-docs/pull/7953/files) with test builds available [here](https://github.com/microsoft/vscode/pull/235232#issuecomment-2615764717). We are hoping this will provide additional time to complete their migration to our list of supported distros without having to disable VSCode updates.

Reviewer 2: legacy server -> linux legacy server

> As we are fast approaching the EOL for [legacy server](https://aka.ms/vscode-remote/faq/old-linux), February 2025 release v1.98 will be the last release supporting these servers. We have explored an option for the SSH extension to let users provide a custom build of glibc and libstdc++ required for connecting to our server. More details on this is documented in this [draft](https://github.com/microsoft/vscode-docs/pull/7953/files) with test builds available [here](https://github.com/microsoft/vscode/pull/235232#issuecomment-2615764717). We are hoping this will provide additional time to complete their migration to our list of supported distros without having to disable VSCode updates.

Reviewer 2: "We are hoping this will provide additional time to complete their migration to our list of supported distros without having to disable VSCode updates."
Let's try to make simpler sentence structure, and have a simple call for action. Maybe something like:
"We hope that this workaround provides additional time to migrate to supported Linux distros without having to disable VS Code updates. If you have any feedback on the workaround flow, do comment [here](https://github.com/microsoft/vscode/issues/231623)."

> As we are fast approaching the EOL for [linux legacy server](https://aka.ms/vscode-remote/faq/old-linux), February 2025 release v1.98 will be the last release supporting these servers. We have explored an option for the SSH extension to let users provide a custom build of glibc and libstdc++ required for connecting to our server. More details on this option is documented in this [draft](https://github.com/microsoft/vscode-docs/pull/7953/files) with test builds available [here](https://github.com/microsoft/vscode/pull/235232#issuecomment-2615764717). We hope that this workaround provides additional time to migrate to supported Linux distros without having to disable VSCode updates. If you have any feedback on the workaround flow, do comment [here](https://github.com/microsoft/vscode/issues/231623).

Reviewer 3: ```suggestion
We are fast approaching the end of support for [Linux legacy server](https://aka.ms/vscode-remote/faq/old-linux). VS Code release v1.98 (February 2025) will be the last release that supports these servers. As of release 1.99, you can no longer connect to these servers.

We have explored an option for the SSH extension where you provide a custom build of glibc and libstdc++ that enables connecting to the server. More details on this option is documented in this [draft](https://github.com/microsoft/vscode-docs/pull/7953/files) with test builds available [here](https://github.com/microsoft/vscode/pull/235232#issuecomment-2615764717).

This workaround should provide additional time for migrating to a supported Linux distro, without having to disable VS Code updates. If you have any feedback on the workaround flow, comment [on this VS Code issue](https://github.com/microsoft/vscode/issues/231623).
```

Reviewer 3: Some rewording. Feel free to tweak.
